cpp/models/lct_secir/initializer_flows.h

```diff
@@ -0,0 +1,251 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Lena Ploetzke
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef LCTSECIR_INITIALIZER_H
+#define LCTSECIR_INITIALIZER_H
+
+#include "memilio/config.h"
+#include "lct_secir/model.h"
+#include "lct_secir/infection_state.h"
+#include "memilio/epidemiology/lct_infection_state.h"
+#include "memilio/math/eigen.h"
+#include "memilio/utils/time_series.h"
+#include "memilio/math/floating_point.h"
+#include "memilio/utils/logging.h"
+#include "memilio/epidemiology/state_age_function.h"
+
+namespace mio
+{
+namespace lsecir
+{
+
+/**
+ * @brief Class that can be used to compute an initialization vector out of flows for a LCT Model.
```

Comment:
I think it makes sense to explain the idea more in detail. For example in the ODE initialization only the flow from C to I is used and here you are using all possible flows. Also, you could mention that you need a certain number of time points that depends on the relevant are of the Erlang density function.

Reply:
Are you okay with this version?